ament_tools/build_types/ament_cmake.py

```diff
@@ -173,8 +183,9 @@ def on_install(self, context):
                 raise VerbExecutionError("Could not find 'msbuild' executable")
             install_project_file = project_file_exists_at(
                 context.build_space, 'INSTALL')
-            yield BuildAction(prefix +
-                              [MSBUILD_EXECUTABLE, install_project_file])
+            if install_project_file is None:
+                raise VerbExecutionError("Could not find INSTALL VS project file.")
```

Comment:
The file is not called `INSTALL`, it's something like `INSTALL.vcproj`, the sentence structure I used was meant to mimic the Unix equivalent, but I'll change it.